test/multest.jl

```diff
@@ -11,6 +11,7 @@ qv = QuArray(v)
 @assert rawcoeffs(qm'*qm') == m*m
```

Comment:
Uh, I guess all of those should be `coeffs` rather than `rawcoeffs`? This would probably also cover the new test case?